client/cmd/dexc-desktop/metainfo/org.decred.dcrdex.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.decred.dcrdex</id>
+  <launchable type="desktop-id">org.decred.dcrdex.desktop</launchable>
+  <name>DCRDEX</name>
+  <summary>Decentralized cryptocurrency exchange</summary>
+  <metadata_license>MIT</metadata_license>
```

Comment:
You'd think if you specified an MIT license here, we would need to have an MIT license somwhere. Can this just be `BlueOak-1.0.0` too, for consistency?

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license